### PR TITLE
ZER-28 Localize main page

### PR DESCRIPTION
**Pull request by haliapats**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by VictoriaBryz — DISMISSED** [review]

File: config/locales/en.yml [hunk 1]
```diff
@@ -30,8 +30,30 @@
...
+      weight: "Baby weight"
+      messages: "Yes, I want to receive email messages with recommendations"
+      result: "CALCULATE RESULT"
+  layouts:
+    footer:
+      ngo: 'ГО "НУЛЬ ВІДХОДІВ ЛЬВІВ"'
+      rights_html: 'Копіювання змісту цього сайту без дозволу правовласника заборонено. Якщо
+        ви хочете скористатися матеріалами з нашого сайту, пишіть на
+        <a href="mailto:hello@zerowastelviv.org.ua"
+          >hello@zerowastelviv.org.ua</a>'
```
Comment (maksymst) [inline config/locales/en.yml:47]: This can stay in html page

File: spec/models/select_spec.rb [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe Select, type: :model do
   subject { create(:select) }
-  LOCALIZATION_PREFIX = "activerecord.errors.models.select.attributes".freeze
+  LOCALIZATION_PREFIX = 'activerecord.errors.models.select.attributes'
```
Comment (maksymst) [inline spec/models/select_spec.rb:7]: what is the purpose of this change?

File: spec/models/value_spec.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 RSpec.describe Value, type: :model do
   subject { create(:value) }
-  LOCALIZATION_PREFIX = 'activerecord.errors.models.value.attributes'.freeze
+  LOCALIZATION_PREFIX = 'activerecord.errors.models.value.attributes'
```
Comment (maksymst) [inline spec/models/value_spec.rb:7]: same as above

**Review by maksymst — CHANGES_REQUESTED** [review]
Plel.ase use more verbose keys for localizations: `welcome_header` instead of `welcome`, `form-description` instead of `question`

File: spec/models/select_spec.rb (hunk 2 above)
Comment (haliapats) [reply spec/models/select_spec.rb:7]: I ran rubocop for the project and it was corrected by it, I will fix it.

File: config/locales/en.yml (hunk 1 above)
Comment (haliapats) [reply config/locales/en.yml:47]: Thanks, done)

File: spec/models/select_spec.rb (hunk 2 above)
Comment (maksymst) [reply spec/models/select_spec.rb:7]: Rubocop removed double quotes. Just add `freeze` again

Comment (haliapats) [reply spec/models/select_spec.rb:7]: Thanks, done)
